src/models.rs

```diff
@@ -33,6 +34,5 @@ pub struct LimitedAccessKey {
 pub struct CreateAccountOptions {
     pub full_access_keys: Option<Vec<PublicKey>>,
     pub limited_access_keys: Option<Vec<LimitedAccessKey>>,
-    pub contract_bytes: Option<Vec<u8>>,
+    pub contract_bytes: Option<Base64VecU8>,
```

Comment:
In order to avoid a breaking change, let’s introduce a new field (`contract_bytes_as_base64_str`) instead of changing the behavior for `contract_bytes`

Reply:
Great! Fixed.